web/modules/custom/dpl_library_agency/src/GeneralSettings.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Drupal\dpl_library_agency;
+
+use Drupal\dpl_react\DplReactConfigBase;
+
+/**
+ * Class that handles general settings.
+ */
+class GeneralSettings extends DplReactConfigBase {
+
+  const EXPIRATION_WARNING_DAYS_BEFORE_CONFIG = 6;
+  const RESERVATION_DETAIL_ALLOW_REMOVE_READY_RESERVATIONS = FALSE;
+  const INTEREST_PERIODS_CONFIG = '180-6 months';
+  const DEFAULT_INTEREST_PERIOD_CONFIG = [
+    "value" => "180",
+    "label" => "6 months",
+  ];
```

Comment:
If you change the language to danish defaults it should be corrected here as well.

Reply:
👍